app/angular/conceptual/conceptual.js

```diff
@@ -385,16 +386,6 @@ const controller = function (ModelAPI, $stateParams, $rootScope, $timeout, $uibM
 			halo.removeHandle('rotate');
 			halo.render();
 		});
-
-		configs.paper.on('link:mouseenter', (linkView) => {
-			const conectionType = ctrl.shapeLinker.getConnectionTypeFromLink(linkView.model);
-			const toolsView = ctrl.toolsViewService.getToolsView(conectionType);
-			linkView.addTools(toolsView);
-		});
-
-		configs.paper.on('link:mouseleave', (linkView) => {
-			linkView.removeTools();
-		});
```

Comment:
I think there is problematic merge around here

Reply:
I'll check, @miltonbsn. Thank you